connectivity/suite.go

```diff
@@ -197,6 +212,64 @@ func Run(ctx context.Context, ct *check.ConnectivityTest) error {
 			return check.ResultOK, check.ResultDrop
 		})
 
+	// This policy denies all ingresses by default
+	ct.NewTest("all-ingress-deny").
+		WithPolicy(denyAllIngressPolicyYAML).
+		WithScenarios(
+			tests.PodToPod(),
+			tests.PodToCIDR(), // This should be allowed by the policy
+		).
+		WithExpectations(func(a *check.Action) (egress, ingress check.Result) {
+			if a.Destination().Address() == "1.0.0.1" || a.Destination().Address() == "1.1.1.1" {
+				return check.ResultOK, check.ResultNone
+			}
+			return check.ResultDrop, check.ResultNone
+		})
+
+	// This policy denies all egresses by default
+	ct.NewTest("all-egress-deny").
+		WithPolicy(denyAllEgressPolicyYAML).
+		WithScenarios(
+			tests.PodToPod(),
+			tests.PodToPodWithEndpoints(),
+		).
+		WithExpectations(func(a *check.Action) (egress, ingress check.Result) {
+			return check.ResultDrop, check.ResultNone
+		})
+
+	// This policy denies all entities by default
+	ct.NewTest("all-entities-deny").
+		WithPolicy(denyAllEntitiesPolicyYAML).
+		WithScenarios(
+			tests.PodToPod(),
+			tests.PodToCIDR(),
+		).
+		WithExpectations(func(a *check.Action) (egress, ingress check.Result) {
+			return check.ResultDrop, check.ResultNone
+		})
+
+	// This policy allows cluster entity
+	ct.NewTest("cluster-entity").
+		WithPolicy(allowClusterEntityPolicyYAML).
+		WithScenarios(
+			// Only enable to local cluster for now due to the below
+			// https://github.com/cilium/cilium/blob/88c4dddede2a3b5b9a7339c1316a0dedd7229a26/pkg/policy/api/entity.go#L126
+			tests.PodToPod(tests.WithDestinationLabelsOption(map[string]string{"name": "other-same-node"})),
```

Comment:
:+1: sorta makes sense but also I think this is effectively API at this point. Ie, we should test that "cluster" does not allow traffic from pods in one cluster to pods in another cluster.

Reply:
good point, I have added the negative test for multi cluster case :100: